arboretum/issue_mgmt/README.md

```diff
@@ -0,0 +1,238 @@
+locker.
+* Behavior: For each Github repository specified, an evidence file is stored in
+the locker containing the issues for that Github repository based on the provided
+search criteria.  TTL is set to 1 day.
+* Configuration elements:
+   * `org.issue_mgmt.github`
+      * Required
+      * `is:issue` is applied to all retrieval configuration to limit retrieval
+      to Github issues by default.  See the `search` option to expand retrieval
+      to pull requests.
+      * List of dictionaries each containing issue retrieval configuration.
+         * `repo`
+            * Required
+            * String in the form of `owner/repo`.
+            * Use to define the Github repository from where issues will be
+            extracted.
+         * `host`
+            * Optional
+            * String in the form of `https://github.my_org.com`.
+            * Defaults to `https://github.com` (public Github)
+            * Use if the repo is a Github Enterprise repo otherwise do not include.
+         * `states`
+            * Optional
+            * List of string in the form of `["open", "closed"]`.
+            * Valid list element values are `"open"` and `"closed"`.
+            * Values are applied as `OR` logic to the search for issues.
+            * Defaults to `["open"]`.
+            * Use if looking to retrieve closed or both open and closed issues
+            otherwise do not include.
+         * `labels`
+            * Optional
+            * Dictionary of string key/value pairs
+               * Valid keys: `"equals"`, `"contains"`, `"startswith"`, and
+               `"endswith"`.
+               * Valid values: List of strings
+               * Each pair is applied as `OR` logic to the search for issues.
+            * Defaults to all labels.
+            * Use if looking to limit issues to issues containing specific labels
+            other do not include.
+         * `search`
+            * Optional
+            * String in the form of a [Github search query string][gh-issues-search].
+               * Use of `search` overrides all other configuration.
+               * `is:issue` is automatically prepended to every search query
+               provided.
+            * Use if looking for finer granularity in your Github issue search
+            criteria otherwise do not include.
+
+* Example configuration:
+
+   ```json
+   {
+     "org": {
+       "issue_mgmt": {
+         "github": [
+           {
+             "repo": "foo-owner/foo-repo",
+             "host": "https://github.foo.com",
+             "states": ["open", "closed"],
+             "labels": {
+               "equals": ["label-one", "label-two"],
+               "contains": ["foo"],
+               "startswith": ["bar"],
+               "endswith": ["baz"]
+             }
+           },
+           {
+             "repo": "bar-owner/bar-repo",
+             "search": "is:open is:closed in:title \"my issue title\""
```

Comment:
Yep, apparently so.  In fact it appears as though you don't need quotes at all because the GH search logic applies OR logic to the words and returns hits if any are found.  I've changed the example to read `"... in:title 'words in my title'".